lib/autsr.gi

```diff
@@ -924,10 +924,10 @@ local ff,r,d,ser,u,v,i,j,k,p,bd,e,gens,lhom,M,N,hom,Q,Mim,q,ocr,split,MPcgs,
     # try to find some further generators
     if Size(AQP)/Size(Aperm)>100 then
       for j in SpecialPcgs(RadicalGroup(AQP)) do
-	if precond(j) then cond(j);fi;
+	cond(j);
```

Comment:
The commit "REBASE: precond does not always work" could/should perhaps be folded into the commit which it modifies? I.e., "ENHANCE: More efforts on automorphism lifting".

I also don't understand what the "REBASE: " prefix is about. If at all, this seems to be a "fixup", nothing is being rebased?

Reply:
REBASE (for me) means exactly what you write. This commit should be squashed it into another commit. (It is only kept separately initially to make it easy to undo if it would cause problems.)